Modules/Options/AdvancedTab/QuestieOptionsAdvanced.lua

```diff
@@ -411,3 +404,23 @@ function QuestieOptions.tabs.advanced:Initialize()
         },
     }
 end
+
+_GetLanguages = function()
```

Comment:
`_GetLanguages` is not defined yet and therefore a global. Just add a `local _GetLanguages` above `QuestieOptions.tabs.advanced:Initialize()`

Reply:
Thanks, fixed